css-three-column-layout/styles.css

```diff
@@ -0,0 +1,50 @@
+body {
+  background-color: #1996da;
+  color: white;
+  font-family: Lato, sans-serif;
+}
+
+p {
+  font-size: 12px;
+  color: rgb(236 236 236);
+  margin: 13px;
+}
+
+img {
+  width: 50%;
+}
+
+.container {
+  max-width: 700px;
+  margin: auto;
+}
+
+.row {
+  display: flex;
+}
+
+.column-full {
+  margin-top: 50px;
```

Comment:
The only styling that should be on a `column` class is `width` or `flex-basis`. Make sure the measurement accurately matches the name of the class as well 🤓 . If you need any other additional styling on top of `width` or `flex-basis`, you should make a separate class that you can add to the element in addition to the `column` class.